FR/2DCode/raw/test_pcr_ok.json

```diff
@@ -25,16 +25,16 @@
     }
   ]
 },
- "CBOR": "A4041A60F2D565061A60A3BB6501624652390103A101A4617481AB6273631B000001797F8BDF18626D616C3344204D65646963696E65736264721B000001797F8BDF28627474684C50363436342D34626E6D71323031392D6E436F562052542D715043526274636E54657374696E672043656E74726562636F6246526263697827646763693A56313A46523A455530535142583651474E493450373052324449454B3442593A31306269736A4C61626F7261746F72796274676938343035333930303662747269323630343135303030636E616DA463666E74664455504F4E4462666E664475706F6E6463676E74654D4152494562676E654D617269656376657265312E302E3063646F626A313936322D30372D3031",
- "COSE": "D2844DA20448D6FC694CB81CEB0B0126A0590116A4041A60F2D565061A60A3BB6501624652390103A101A4617481AB6273631B000001797F8BDF18626D616C3344204D65646963696E65736264721B000001797F8BDF28627474684C50363436342D34626E6D71323031392D6E436F562052542D715043526274636E54657374696E672043656E74726562636F6246526263697827646763693A56313A46523A455530535142583651474E493450373052324449454B3442593A31306269736A4C61626F7261746F72796274676938343035333930303662747269323630343135303030636E616DA463666E74664455504F4E4462666E664475706F6E6463676E74654D4152494562676E654D617269656376657265312E302E3063646F626A313936322D30372D30315840FD1A9BC486592856D01500D4D7366B6D5131F8CF776BAD7C828AC0263BF9D1AE8C226003FDE396AAB52C40E453C3F31421E6D1FD8B343261CCA925E5E6A50A26",
```

Comment:
This COSE payload is not compliant with expected JSON Schema.
Value of dr and sc should not be int but date value in ISO 8601 format.

![image](https://user-images.githubusercontent.com/84303320/118812067-500ac400-b8e0-11eb-8b2f-be248a6539b4.png)

Reply:
fix ok